test/mail_helper.js

```diff
@@ -53,7 +53,15 @@ module.exports = (printLogs) => {
               smsLink = smsUrlMatch && smsUrlMatch[1]
             }
 
-            var name = emailName(mail.headers.to)
+            var name
+            if (mail.headers['x-email-service'] === 'fxa-email-service' && mail.headers.to.charAt(0) === '<') {
```

Comment:
@brizental I think we can just do something like:

```js
'<emailservice.wat@restmail.net>'.replace(/\<(.*?)\>/g, '$1');
```

 on the `mail.headers.to`

Reply:
Yeah, so, that's what I did at first, but then I thought, what if we wanted the `<` or `>`, now I realize that's unlikely and also the way I do it doesn't prevent for this case... lol

Anyways, I'll change it.